conans/client/build/compiler_id.py

```diff
@@ -0,0 +1,210 @@
+import os
+import tempfile
+from six import StringIO
+
+from conans.client.runner import ConanRunner
+from conans.model.version import Version
+
+
+GCC = "gcc"
+LLVM_GCC = "llvm-gcc"  # GCC frontend with LLVM backend
+CLANG = "clang"
+APPLE_CLANG = "apple-clang"
+SUNCC = "suncc"
+MSVC = "Visual Studio"
+INTEL = "intel"
+QCC = "qcc"
+
+
+class CompilerId:
```

Comment:
So far, all of our classes extend "object" (py2)

Reply:
changed to use object